specification/maps/data-plane/Common/preview/1.0/common.json

```diff
@@ -614,6 +614,7 @@
     },
     "View": {
       "name": "view",
+      "x-ms-client-name": "userRegion",
       "in": "query",
```

Comment:
This should probably incorporate the enum described in https://github.com/Azure/azure-rest-api-specs/pull/16283

Reply:
Incorporated the changes from #16283. It's now an enum.